src/statusChange.js

```diff
@@ -0,0 +1,12 @@
+export default function toggleTodo(id, todos) {
+  const newtodos = todos.map((e) => {
+    if (e.index === Number(id)) {
+      e.completed = !e.completed;
+      return e;
+    }
+    return e;
+  });
+  window.localStorage.setItem('todos', JSON.stringify(newtodos));
+  // eslint-disable-next-line
+  location.reload();
```

Comment:
So with these pieces of code, once the user checked an item, the page seems to reload, that is not what we wanted right.

we want a user to check any item and our localStorage should save the changes.

Reply:
Hi,

Thanks for your comment. I updated localstorage with the line:

  window.localStorage.setItem('todos', JSON.stringify(newtodos));
   
The location.reload() part was just for better user experience.